### PR TITLE
7.2 User | Custom Filters | LRDOCS-6902

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-6902, read the article until the STOP message, please. The rest is copied instructions from the LPS story ticket, which I plan to turn into how-to docs, since this is a complex feature. Let me know what you think of this article, and the plan to create a couple of use-case articles.

### CODE REVIEW

**Comment by rbohl** [comment]
Jira is down for maintenance right now, will update the ticket later.

**Comment by lipusz** [comment]
**Started reviewing...** :mag::nerd_face:

File: user/articles/120-search/03-custom-filters.markdown [hunk 1]
```diff
@@ -0,0 +1,360 @@
+---
+header-id: custom-filters
+---
+
+# Custom Filters
```
Comment (lipusz) [inline user/articles/120-search/03-custom-filters.markdown]: Change: I'd like to rename it to "Custom Filter Search Widget" or "widgets" to distinguish the article from the "search framework filters" as well as keep in the scope of "search widgets".

**Comment by lipusz** [comment]
> do you know if this widget works with Solr, too?

Answer: yes, it should work, because we have a translator for the listed query types in the Solr modules as far as I can see.

**Comment by lipusz** [comment]
Looks like it does not work with Solr currently, because https://github.com/brianchandotcom/liferay-portal/pull/70955 added the necessary changes to the elasticsearch 6 module only: https://github.com/brianchandotcom/liferay-portal/pull/70955/commits/82cd80580e377e951645726ff9b0e5be370f4b69

**Comment by lipusz** [comment]
Looks good, see my comments above. :+1:

File: user/articles/120-search/03-custom-filters.markdown (hunk 1 above)
Comment (rbohl) [reply user/articles/120-search/03-custom-filters.markdown]: > # Filtering Search Results with the Custom Filter Widget

What about that? @lipusz

Comment (lipusz) [reply user/articles/120-search/03-custom-filters.markdown]: :+1: Sounds good to me.

File: user/articles/120-search/03-custom-filters.markdown [hunk 2]
```diff
@@ -1,23 +1,21 @@
----
-header-id: custom-filters
----
+# Filtering Search Results with the Custom Filter Widget
 
-# Custom Filters
+| **This widget only works with Elasticsearch.**
```
Comment (lipusz) [inline user/articles/120-search/03-custom-filters.markdown]: Remove: let's remove this: currently there is not Solr connector for 7.2 released, so technically it's up to the team to add support. We can add it back if we won't make it happen.

**Comment by rbohl** [comment]
Forwarded: https://github.com/sez11a/liferay-docs/pull/4202

**Pull request closed without merging**